Azure-Samples/Cognitive-Services-Voice-Assistant
# User/travisw/correct uwpvas build configurations

## Purpose
The new UWPVoiceAssistantSample project inherited some old optimizations to one of its .csproj files to avoid accidentally compiling in ARM. That platform is relevant now *and* it's still in the .sln file, which means:

* Right now, attempting to build in ARM or ARM64 (which are first in the list and default selected because of VS having alphabetical platform target rules) will fail, complaining about missing metadata files or architecture mismatch (since the main project is trying to build x86)
* There's no way without extra work to build for ARM/ARM64 (and you can run this on ARM/ARM64 devices, e.g. HL2)

This corrects the build configurations to make all target platforms ({Debug,Release}|{ARM,ARM64,X64,X86}) build.


## Does this introduce a breaking change?
```
[ ] Yes
[X] No
```

## Pull Request Type
```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Clone/open the project from master on a fresh machine (or delete the .vs folder and all four bin/obj folders for a clean state)
* Open the .sln and observe the default configuration is Debug|ARM; try to build it and it'll break
* Look into the platform configuration and see that the main project only has X86 and X64 available

Then

* Repeat the steps in the topic branch/with the change applied
* Observe that ARM/ARM64 build in addition to X86/X64
